### ISSUE TITLE
Sponsor Badge

### ISSUE THREAD

**Issue by holtwick** [framework7io/framework7#3257]
Put a sponsor badge on your GitHub page ;)

<img width="190" alt="image" src="https://user-images.githubusercontent.com/49557/61523275-4bd79c00-aa14-11e9-8289-1f50462b5f81.png">

It is as easy as adding `.github/FUNDING.yml` to the project with this content:

```
patreon: vladimirkharlampidi
```

This should make it easy to discover how to support you :)

**Comment by nolimits4web** [comment]
Thanks for the tip :)

**Comment by nolimits4web** [comment]
And thank you for your support! 🙌

**Comment by holtwick** [comment]
Sure, your work is awesome! It is the foundation of the iOS version of https://onepile.app/ and works perfectly. I'm usually doing native UI, but your framework is so production ready that I decided to switch: https://holtwick.de/blog/2019-03-14-hybrid-apps And instead of Cordova I use my own wrapper both on iOS and macOS and so far it worked great https://holtwick.de/blog/2018-12-20-web-technologies. Finally, I can leave some more of the unsexy Objective-C and Swift code behind ;)  Thanks for all the work you have put into it!

**Comment by nolimits4web** [comment]
Awesome, congrats! I believe you need to post it on Framework7 showcase. By the way, i see you also have desktop and web version. So just wondering what UI do you use there? Framework7 didn't fit there?

**Comment by holtwick** [comment]
> Awesome, congrats! I believe you need to post it on Framework7 showcase. 

I did under the former name "Collect". I will update the entry in a minute.

> By the way, i see you also have desktop and web version. 

I'm using [Buefy/Bulma](https://buefy.org/documentation) for it with some custom CSS. It works great with Vue.js and saves some work there. Since I'm sharing a lot of code and modules, I might change that in future to not have to implement stuff twice. But I'm not yet sure how to do that.

> So just wondering what UI do you use there? Framework7 didn't fit there?

At that time you didn't have the Aurora theme yet. And the UI was quite mobile specific. For the UI on desktop I wanted something lighter and more flexible. While it was a work in progress, I didn't know where to place everything on the screen. I might revisit the decision in the future. 

By the way, for the rich text editor I use [TipTap](https://tiptap.scrumpy.io/). I read you are going to implement something like this for Framework7 as well. Probably you want to keep it much simpler and agnostic of Vue.js etc. But this would probably most high-end component in general to use for such a purpose. I implemented a lot of `contentEditable` editors myself and I'm afraid you are opening the box of Pandora 😉
